### PR TITLE
Add and Remove

### PR DESCRIPTION
**Pull request by phelian23**
Add the  CRUD (create, update, delete) methods

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
```
Comment (sinansevgi) [inline README.md:17]: - [ ] You should create a new JavaScript file for the new functionality.

File: README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 ## Application Page url
```
Comment (sinansevgi) [inline README.md:19]: - [ ] Users should be able to edit task descriptions. I couldn't find any function related to this.

File: README.md [hunk 3]
```diff
@@ -17,7 +17,7 @@
...
 
-https://phelian23.github.io/to-do-list/
+Here is a link to the [live](https://phelian23.github.io/to-do-list/) demo
```
Comment (sinansevgi) [inline README.md:21]: [Optional]

- [ ] You should update the DOM elements instead of refreshing the page after every operation.

File: README.md [hunk 4]
```diff
@@ -17,7 +17,7 @@
...
 
 
 ## Getting Started
```
Comment (sinansevgi) [inline README.md:23]: Note for future projects:
**You must ensure that your pull request includes all commits related to the change you made. We are using the `Files Changed` feature of GitHub to see which files are updated between pull requests.**

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @phelian23, 

Good job so far!
`Clean all completed` function works well :+1: 
There are some issues that you still need to work on to go further but you are almost there!

 ## Required Changes ♻️
_Please check the inline comments for my suggestions_ :arrow_down:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (DeliceLydia) [reply README.md:19]: Addressed this

File: README.md (hunk 3 above)
Comment (DeliceLydia) [reply README.md:21]: resolved this

File: src/change.js [hunk 5]
```diff
@@ -1,11 +1,17 @@
...
+    } else {
+      listArray[i].completed = true;
+      descript.style.textDecoration = 'line-through';
+    }
+  } else if (listArray[i].completed === true) {
+    listArray[i].completed = false;
+    descript.style.textDecoration = 'none';
+  } else {
+    listArray[i].completed = true;
+    descript.style.textDecoration = 'line-through';
```
Comment (DeliceLydia) [inline src/change.js:15]: - [ ] Deleting a task function is missing please work on that

File: README.md (hunk 3 above)
Comment (DeliceLydia) [reply README.md:21]: - [ ] Fix live demo link

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @phelian23, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/change.js (hunk 5 above)
Comment (phelian23) [reply src/change.js:15]: the deleting function is not missing, it's in the task.js file, and it's working

**Review by EricMbouwe — APPROVED** [review]
### Hi @phelian23, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**